### DISCUSSION TITLE
Need help to implement scroll on the webpage

### DISCUSSION THREAD

**Discussion by assopri** [cefsharp/CefSharp#4003] [General]
Hello! Trying to evaluate the following script to scroll:
Browser.EvaluateScriptAsync("window.scroll(0,"+ times + ");");

However it doesn`t imitate scroll, for instance here https://www.mvideo.ru/televizory-i-cifrovoe-tv-1/televizory-65

Any suggestions how to implement scroll?

**Comment by amaitland** [comment]
> However it doesn`t imitate scroll

Can you clarify what you mean by this? What behaviour are you hoping to see exactly? Are you trying to scroll to a specific element? Scroll by a set number of lines?

There are a few options including using  SendMouseWheelEvent. Just depends on the behaviour you are trying to implement. 

http://cefsharp.github.io/api/97.1.x/html/M_CefSharp_WebBrowserExtensions_SendMouseWheelEvent_2.htm

**Reply by assopri** [reply]
Cool! That was what I need. What about scrolling to the specific element? Any outofbox solutions?

**Reply by amaitland** [reply]
Executing JavaScript to call https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView is likely the easiest option.
